### DISCUSSION TITLE
403: You cannot create a playlist for another user

### DISCUSSION THREAD

**Discussion by qweered** [Chimildic/goofy#165] [Помощь с ошибкой] [answered]
Начал устновку, дошел по инструкции до 4 шага создания плейлиста и появилась ошибка при запуске createFirstPlaylist

![chrome_uuIFJILCgT](https://user-images.githubusercontent.com/41731334/148550780-e9685615-43f3-4b4f-9d5b-0c60a5c5e0e9.png)

Права доступа обновил как было сказано, ошибка осталась

16:24:20	Ошибка	Номер: 403
Адрес: https://api.spotify.com/v1/users/undefined/playlists
Текст ответа: {
  "error" : {
    "status" : 403,
    "message" : "You cannot create a playlist for another user."
  }
} 

Описание и решение ошибок: https://chimildic.github.io/goofy/#/errors
16:24:02	Ошибка	
TypeError: Cannot read property 'id' of undefined
saveAsNew	@ library.gs:1898
saveWithModify	@ library.gs:1938
saveWithReplace	@ library.gs:1913
createFirstPlaylist	@ main.gs:16

**Comment by Chimildic** [comment, marked as answer]
Баг. Обновите весь код файла `library` на [исправленное](https://script.google.com/d/1DnC4H7yjqPV2unMZ_nmB-1bDSJT9wQUJ7Wq-ijF4Nc7Fl3qnbT0FkPSr/edit?usp=sharing).
